src/main/webapp/WEB-INF/views/cars/drivers.jsp

```diff
@@ -0,0 +1,21 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Car drivers editor</title>
+</head>
+<body>
+<h1>Car drivers</h1>
+car id: ${car.id}<br>
+car model: ${car.model}<br>
+car manufacturer: ${car.manufacturer.name}<br>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers">
+    <c:forEach items="${drivers}" var = "driver">
+        <input type="checkbox" id="${driver.id}" name="${driver.id}" <c:if test="${car.drivers.contains(driver)}">checked</c:if>>
+        <label for="${driver.id}">${driver.name}: ${driver.licenseNumber}</label><br>
+    </c:forEach>
+    <input type="hidden" name="id" value="${car.id}" />
+    <button type="submit">Confirm</button>
+</form>

```

Comment:
what is this page for? diplaying drivers of car is not in this task) especially you don't need any form)

Reply:
it is about the last TODO things in the readme file
"Create a new controller: Implement adding driver to car."
and when I see this picture in common mistake 
![image](https://user-images.githubusercontent.com/103377753/177129988-8277d14f-915c-48fb-b646-200b55aaf5fa.png)
I try to create a page with the ability to change drivers in the car.
In my opinion, this page is more useful than 3 different pages where you can see all drivers, add only one or remove only one
Plus I think it improves performance because when we update a car we delete all drivers and then add all driver to DB